frontend/src/views/contact-form.ts

```diff
@@ -69,33 +70,32 @@ export class ContactForm extends LitElement {
       ></vaadin-combo-box>
 
       <div class="buttons">
-        <vaadin-button @click=${this.save} theme="primary">Save</vaadin-button>
-        <vaadin-button @click=${this.delete} theme="error"
+        <vaadin-button @click=${this.save} theme="primary" ?disabled="${this.binder.invalid || this.binder.submitting}">Save</vaadin-button>
+        <vaadin-button @click=${this.delete} theme="error" ?disabled="${!this.binder.value && this.binder.value}"
```

Comment:
Isn't this always false `!this.binder.value && this.binder.value` ? (It's basically testing if `this.binder.value` is falsy and truthy at the same time, right?)
Should this be something else? Maybe `?disabled="${!this.binder.value || this.binder.submitting}"`?

Reply:
Good catch, my initial attendtion was to have sth like `?disabled="${!this.binder.value && this.binder.value.isPersisted}`
But `isPersisted` is not available, I remove that property but forgot the update the whole condition. :D

We can just use `!this.binder.value` I think.